code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm

```diff
@@ -375,15 +389,40 @@ Difficulty: Medium
 	if(!lava_success)
 		arena_escape_enrage()
 
-/mob/living/simple_animal/hostile/megafauna/dragon/AltClickOn(atom/movable/A)
-	if(!istype(A))
+/mob/living/simple_animal/hostile/megafauna/dragon/death()
+	QDEL_NULL(internal) // so drake corpses don't have a gps signal
+	. = ..()
+
+/mob/living/simple_animal/hostile/megafauna/dragon/ex_act(severity, target)
+	if(severity == 3)
```

Comment:
is this not defined?

Reply:
it was just weird git stuff, i removed the dragon alt click and moved it to lesser dragon since it wasnt the actual dragon swoop it was a snowflaked version.

Reply:
when I say defined I mean the severity numbers like an `EXPLOSION_HEAVY` or something.

doesn't look like one exists tho so *shrug*